ERC721/guide.md

```diff
@@ -0,0 +1,85 @@
+# NFT Contract Interaction Guide
+
+- Start API server ``logiclab start -d ~/.coco/lab``
+- ```cd ERC721/```
+- ```coco lab start``` Cocolab REPL start
+- ```compile NFT```
+- ```get NFT``` lists alll endpoints
+- ```register username```
```

Comment:
You can register with a fixed address, so don't have to change the addresses below to match the current username. Is it possible to add some memory variables, so you can e.g. use "Me", "Friend" in the commands instead of those long addresses?

Reply:
Ok, if "username's" address is never used, then fine, we can just leave everything as it is.